docs/voice/call-session-notifications.md

```diff
@@ -0,0 +1,176 @@
+# Call Session Notification
```

Comment:
Do we want to introduce an additional definition of described event type? "CSN" is an internal name of the feature and it's not a part of API Reference now. So I suggest either keep only Telephony Sessions Notifications (TSN) or update API Ref with knowledge of what CNS means.